app/controllers/activities_controller.rb

```diff
@@ -4,6 +4,25 @@
 class ActivitiesController < AuthenticatedController
   include ProjectScoped
 
+  def index
+    activities = Activity.includes(:trackable)
+                         .order(created_at: :desc)
+                         .page(params[:page])
+
+    @page = params[:page].present? ? params[:page] : 1
+
+    @activities_groups = activities.group_by do |activity|
+      activity.created_at.strftime(Activity::ACTIVITIES_STRFTIME_FORMAT)
+    end
+
+    respond_to do |format|
+      format.html
+      format.js do
+        render 'activities/_activities_groups.html.erb', locals: { activities_groups: @activities_groups }
```

Comment:
Lets use the rails convention and have an `index.js.erb` template please. We can remove this entire block and handle the render logic in the view.

Otherwise we're using `format.js` to render a `.html.erb`. We should probably also javascript-scape the result. There are other examples throughout the code where we've followed this pattern.

Reply:
Done, much cleaner now :+1: